internal/wasm/sdk/opa/opa_test.go

```diff
@@ -206,6 +207,18 @@ a = "c" { input > 2 }`,
 				{Result: `{{"y": 2, "x": "q"}}`},
 			},
 		},
+		{
```

Comment:
I'll drop this test case, these have their package names hardcoded to `p`; it doesn't help us.